samples/dotnet_core_managed_gateway/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+using Microsoft.Azure.Devices.Gateway;
+
+namespace dotnet_core_managed_gateway
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                if (args.Length < 1 || File.Exists(args[0]) == false)
```

Comment:
Cool! Thanks! 

---
In reply to: [105793096](https://github.com/Azure/azure-iot-gateway-sdk/pull/163#discussion_r105793096) [](ancestors = 105793096)